VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationPreference.java

```diff
@@ -34,23 +44,40 @@ public EdxLocalParseChannel getByChannelId(String channelId){
         return null;
     }
 
-    public void removeByCourseId(String courseId){
+    /**
+     *
+     * @param courseId
```

Comment:
Are these comments being added automatically? This one doesn't seem very useful.